lib/jekyll/commands/serve/live_reload_reactor.rb

```diff
@@ -68,11 +68,11 @@ def start(opts)
         # http://feedback.livereload.com/knowledgebase/articles/86174-livereload-protocol
         def reload(pages)
           pages.each do |p|
-            json_message = JSON.dump({
+            json_message = JSON.dump(
               :command => "reload",
               :path    => p.url,
-              :liveCSS => true,
-            })
+              :liveCSS => true
```

Comment:
This will result in mangled Git history when something new is added below, but 🤷‍♂️

Reply:
Oooh, I really prefer the dangling , tbh. What’s the reasoning to remove the dangling comma?

Reply:
I applied Rubocop's default cop: http://rubocop.readthedocs.io/en/latest/cops_style/#stylebracesaroundhashparameters